srv/modules/runners/push.py

```diff
@@ -132,6 +152,56 @@ def output(self, common):
                 custom = self.pillar_dir + "/" + default_path
                 self._custom(custom)
 
+    def convert(self, common):
+        """
+        Process all hardware profiles
+        """
+        for pathname in common.keys():
+            for filename in common[pathname]:
+                if 'profile-' in filename:
+                    with open(filename, "r") as yml:
+                        content = yaml.safe_load(yml)
+                    migrated = self._migrate(content)
+                    newfilename = re.sub('profile-', 'migrated-profile-', filename)
+                    path_dir = os.path.dirname(newfilename)
+                    _create_dirs(path_dir, self.pillar_dir)
+                    with open(newfilename, "w") as yml:
+                        yml.write(yaml.dump(migrated, Dumper=self.friendly_dumper,
+                                                  default_flow_style=False))
+
+    def _migrate(self, yaml):
+        """
+        Migrate the original data structure to the ceph namespace data
+        structure
+        """
+        if 'storage' in yaml and 'osds' in yaml['storage']:
+            yaml['ceph'] = {}
+            yaml['ceph']['storage'] = {}
+            yaml['ceph']['storage']['osds'] = {}
+            for osd in yaml['storage']['osds']:
+                yaml['ceph']['storage']['osds'][osd] = { 'format': 'bluestore' }
```

Comment:
Although most of the time correct, we change the format to bluestore regardless of the user's choice. Same with the wal/db distrubution to the old journal.